lib/av_client.ts

```diff
@@ -396,9 +396,9 @@ export class AVClient implements IAVClient {
 
       let encryptedAffidavit;
 
-      if (affidavit && this.latestConfig && this.latestConfig.castRequestItemAttachmentEncryptionKey) {
+      if (affidavit && this?.latestConfig?.items?.electionConfig?.content?.castRequestItemAttachmentEncryptionKey) {
```

Comment:
Given we're using TypeScript, can we make these `null` assertions at the Type level? (i.e., fail when parsing the API response)

Reply:
@MateuszMichalowski @unnunhexium WDYT?

Reply:
Are you referring to add union types with null possibility to these @av-lukas or you have something else in mind?

Reply:
I'm not sure how union types would be used here, but I'm referring to the fact that at this point in the lifecycle of the av_client, if the affidavit is present, then it also means that the client has been told to ask for one, via the configuration object. So, in that case, the `castRequestItemAttachmentEncryptionKey` should always be present, and the null checks shouldn't be necessary. The class that wraps it should enforce it to be present when parsing the request.